markdown/opentelemetry-intro/opentelemetry-intro.md

```diff
@@ -0,0 +1,320 @@
+author: Jesse Menning
+summary:
+id: opentelemetry-intro
+tags:
+categories: opentelemetry, nyctaxi
```

Comment:
can you please change the category here to `taxi` instead of `nyctaxi`, we will be updating the cats of all codelabs related to the nyc taxi demo to include that tag so we can organize them better